doc/release-notes/6980-datacite-reservation-upgrade.md

```diff
@@ -0,0 +1,26 @@
+## Dataverse installations using Datacite: upgrade action required.
+
+For installations that are using Datacite, Dataverse v5.0 introduces a change in the process of registering the Persistent Identifier (DOI) for a dataset. Instead of registering it when the dataset is published for the first time, Dataverse will try to "reserve" the DOI when it's created (by registering it as a "draft", using Datacite terminology). When the user publishes the dataset, the DOI will be publicized as well (by switching the registration status to "findable"). 
+
+This approach makes the process of publishing datasets simpler and less error-prone. If the DOI has not been reserved (for example, if Datacite was down when the dataset was created), the author can still attempt to publish. But it does help to ensure that the DOI is guaranteed to be available beforehand. New APIs have been provided for finding any unreserved Datacite-issued DOIs in your Dataverse, and for reserving them (see below). If you are upgrading an installation that uses Datacite, we specifically recommend that you reserve the DOIs for all your pre-existing unpublished drafts as soon as Dataverse v5.0 is deployed, since none of them were registered at create time. 
```

Comment:
The sentence starting with "But" feels unnecessary.

```suggestion
This approach makes the process of publishing datasets simpler and less error-prone. If the DOI has not been reserved (for example, if Datacite was down when the dataset was created), the author can still attempt to publish. New APIs have been provided for finding any unreserved Datacite-issued DOIs in your Dataverse, and for reserving them (see below). If you are upgrading an installation that uses Datacite, we specifically recommend that you reserve the DOIs for all your pre-existing unpublished drafts as soon as Dataverse v5.0 is deployed, since none of them were registered at create time.
```

Reply:
But, without the sentence starting with "But", doesn't it become completely unclear why they need to bother at all? 
Or do you feel it's explained enough by the first sentence ("simpler and less-error prone")?

Reply:
Yeah, I was thinking it's explained enough.